Foundation/NSCoder.swift

```diff
@@ -14,6 +14,7 @@ extension NSCoder {
     /// supports exceptions here, and there may be other approaches supported
     /// in the future, so its included for completeness.
     public enum DecodingFailurePolicy : Int {
+        case raiseException
```

Comment:
Adding a new case here is potentially a backwardly incompatible API change, since now raiseException will inherit the value previously used for setErrorAndReturn

Reply:
Fair point, didn't realize SwiftFoundation had shipped with ABI compatibility guarantees. Feel free to swap the enum cases around (which will be different to Darwin, but that doesn't really matter I guess). Or remove it.

Reply:
We haven't yet, but adding an enum case to the end is the kind of habit that's good to get used to ;)

Reply:
In the absence of ABI stability, I think we should prefer [consistency with Darwin](https://developer.apple.com/documentation/foundation/nscoder.decodingfailurepolicy/1643150-raiseexception).

Reply:
👍🏻